src/storage/src/render/sources.rs

```diff
@@ -457,7 +454,16 @@ where
                                                 as_of_frontier,
                                                 SourceInstanceDesc {
                                                     description: tx_src_desc,
-                                                    storage_metadata,
+                                                    storage_metadata: CollectionMetadata {
```

Comment:
I know this looks goofy but there's not currently a way to access the shard_id directly from the controller here (and we probably don't want that?).  Open to suggestions.  At the very least, I'll change the `unwrap` to an `expect`

Reply:
okay